parser.y

```diff
@@ -11154,21 +11156,36 @@ CreateBindingStmt:
 	{
 		startOffset := parser.startOffset(&yyS[yypt-2])
 		endOffset := parser.startOffset(&yyS[yypt-1])
-		//setOprStmt := $5.(*ast.SetOprStmt)
 		setOprStmt := $5
 		setOprStmt.SetText(strings.TrimSpace(parser.src[startOffset:endOffset]))
 
 		startOffset = parser.startOffset(&yyS[yypt])
-		//hintedSetOprStmt := $7.(*ast.SetOprStmt)
 		hintedSetOprStmt := $7
 		hintedSetOprStmt.SetText(strings.TrimSpace(parser.src[startOffset:]))
 
 		x := &ast.CreateBindingStmt{
+			BindingTp:   ast.BindingForStmt,
 			OriginNode:  setOprStmt,
 			HintedNode:  hintedSetOprStmt,
 			GlobalScope: $2.(bool),
 		}
 
+		$$ = x
+	}
+|	"CREATE" GlobalScope "BINDING" "FOR" "DIGEST" stringLit "USING" hintComment
```

Comment:
can uses create a biding with:

```sql
create binding for
	select ...
using
	/*+ USE_INDEX(@`sel_1` `mysql`.`t` ) */
```

Reply:
This is a good idea to complement the syntax. However it is not a required feature to make it available in UI. I guess SQL binding may be less frequently used when there is a UI for it. Maybe we can defer it until necessary?